RusimbiP/Teamwork
# #168977551 Configure Database

#### What does this PR do?
Have view database configured
#### Description of Task to be completed?
Have the following actions performable:
- Create Tables, 
- Drop tables,
- Insert initial data into the database
#### How should this be manually tested?
After cloning this project cd into it and run **npm i** to install dependencies

- Using SQL shell or pgAdmin create a database
- in .env file, create a DATABASE_URL following this format 
`postgres://{db_username}:{db_password}@{host}:{port}/{db_name}`

- run `npm run up` to check if tables can be created
- run `npm run seed` to check if data can be seeded into the database
- run `npm run down` to check if tables can be deleted/dropped
- run `npm run migrations` to check if tables can be deleted/dropped, created and have data inserted

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#168977551 
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
